superset/connectors/sqla/views.py

```diff
@@ -167,9 +170,7 @@ class TableModelView(SupersetModelView, DeleteMixin):  # noqa
         'schema': _(
             "Schema, as used only in some databases like Postgres, Redshift "
             "and DB2"),
-        'description': Markup(
-            "Supports <a href='https://daringfireball.net/projects/markdown/'>"
-            "markdown</a>"),
+        'description': _("Supports markdown"),
```

Comment:
I removed the `utils.markdown()` of `expression` column, but kept the `utils.markdown()` of `d3format` column:
<img width="1489" alt="2017-06-22 00 02 38" src="https://user-images.githubusercontent.com/7597155/27394304-44fb85da-56de-11e7-8789-dea990d4d22c.png">